bin/BuildPackages.sh

```diff
@@ -85,6 +98,7 @@ make TOPDIR=`pwd`
 chmod -R a+rX .
 cd bin
 aa=`./config.guess`
+# TODO dangerous to parse output of ls, use find instead
```

Comment:
The reason I did not resolve this is because I did not check properly how to build carat. It failed on my system, and did not investigate further, yet.